htdocs/compta/facture/card.php

```diff
@@ -4657,8 +4657,7 @@ function setRadioForTypeOfInvoice() {
 	}
 	$morehtmlref .= '</div>';
 
-	$object->totalpaid = $totalpaid; // To give a chance to dol_banner_tab to use already paid amount to show correct status
-
+	$object->totalpaid = $totalpaid + $totalcreditnotes + $totaldeposits; // To give a chance to dol_banner_tab to use already paid amount to show correct status
```

Comment:
For consistency of meaning of variable, totalpaid must remain to amount of payments only.
However, I initialize the other properties ->totalcreditnote, ->totaldeposits and ->remaintopay...
Is it enough ?

Reply:
I used these properties in CommonInvoice::getLibStatut() to get the good invoice status. Is it OK ?

Reply:
Things are not clear at a lot of place of code.
To I pushed a change to standardize the code. The amount already paid in parameter of getLibStatus should be already consolidated (amount of payment + credit notes + deposit). it was sometimes and sometimes not.

Reply:
I create another PR: https://github.com/Dolibarr/dolibarr/pull/31300.
Your commit does not fix the status problem when the status is displayed with dol_banner_tab() function. My PR fix that.